OpenRA.Game/Server/Server.cs

```diff
@@ -856,17 +859,14 @@ public void DispatchOrdersToClients(Connection conn, int frame, byte[] data)
 
 		void RecordOrder(int frame, byte[] data, int from)
 		{
-			if (recorder != null)
-			{
-				recorder.ReceiveFrame(from, frame, data);
+			recorder?.ReceiveFrame(from, frame, data);
```

Comment:
this is probably ok, but,
this doesn't change functionality?
i.e. recorders it always set prior if needed?
i.e. data below can safely be handled even if recorder is null?

Reply:
having the code below run unconditionally is what matters here